src/traces/tracePlot.component.tsx

```diff
@@ -54,6 +54,12 @@ const TracePlot = (props: TracePlotProps) => {
           type: 'linear',
           exponentformat: 'none',
           automargin: true,
+          tickformat: '.4g',
```

Comment:
The tickformat: '.4g' ensures that x-axis values are displayed with a maximum of 4 significant figures. The tickformatstops allow finer control over precision in different numerical ranges:


-    [0, 0.01] → Uses '.2g' for higher precision in very small values.
-    [0.01, 1] → Uses '.3g' for moderate precision in small values.
-    [1, null] → Uses '.4g' to maintain standard formatting for larger values.
-    This approach ensures clarity while avoiding excessive decimal places or scientific notation where unnecessary.

Reply:
Change it to a global one, remove tick format.

Reply:
remove and put in a separate pr